fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Request.kt

```diff
@@ -34,15 +37,6 @@ class Request(
     var timeoutInMillisecond: Int,
     var timeoutReadInMillisecond: Int
 ) : Fuel.RequestConvertible {
-
-    @Deprecated(replaceWith = ReplaceWith("method"), message = "http naming is deprecated, use 'method' instead")
-    val httpMethod
-        get() = method
-
-    @Deprecated(replaceWith = ReplaceWith("headers"), message = "http naming is deprecated, use 'headers' instead")
```

Comment:
should we be removing these now?

Reply:
@kittinunf

Reply:
Since this is a breaking change, let's do it 🎉.

Perhaps, this will be a major bump for the next version, do you 2 think it makes sense?

Reply:
Yes!